xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_support.xsd

```diff
@@ -116,6 +119,60 @@ Rail transport, Roads and Road transport
 			</xsd:restriction>
 		</xsd:simpleContent>
 	</xsd:complexType>
+	<xsd:simpleType name="DoorTypeEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed values for DOOR TYPE +2.0</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="xsd:NMTOKEN">
+			<xsd:enumeration value="hingedSingle">
+				<xsd:annotation>
+					<xsd:documentation>Single door hinged at one side.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="hingedPair">
+				<xsd:annotation>
+					<xsd:documentation>Pair of doors, hinged at either side of ENTRANCE.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="slidingSingle">
+				<xsd:annotation>
+					<xsd:documentation>Single door, sliding to one side.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="slidingPair">
+				<xsd:annotation>
+					<xsd:documentation>Pair of sliding doors, sliding apart to each side.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="foldingSingle">
+				<xsd:annotation>
+					<xsd:documentation>Single door, folding to one side.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="foldingPair">
+				<xsd:annotation>
+					<xsd:documentation>Pair of folding doors, folding to each side.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="hingedRamp">
+				<xsd:annotation>
+					<xsd:documentation>Door hinged at bottom to drop down as boarding ramp.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="revolving">
+				<xsd:annotation>
+					<xsd:documentation>Revolving Door.</xsd:documentation>
```

Comment:
Please add a description, and lowercase door.

Reply:
changed. However a revolving door is self-explanatory.